.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,45 @@
+** DO NOT ERASE THESE INSTRUCTIONS WITHOUT READING THEM FIRST! **
+
```

Comment:
I would put the help instructions here at the top, it’s more welcoming.

Reply:
Done